src/Mod/Arch/ArchWall.py

```diff
@@ -1340,9 +1343,9 @@ def getExtrusionData(self,obj):
                                 #    dvec2 = DraftVecUtils.scaleTo(dvec,off)
                                 #    wire = DraftGeomUtils.offsetWire(wire,dvec2)
 
-                                # Get the 'offseted' wire taking into account
-                                # of Width and Align of each edge, and overall
-                                # Offset
+                                ## Get the 'offseted' wire taking into account
+                                ## of Width and Align of each edge, and overall
+                                ## Offset
```

Comment:
Do not add double hash signs `##`, these are interpreted in a special way by Doxygen (documentation builder). We need a simple hash for comments.

Reply:
Thanks for checking, reverted the changes.